common/types.go

```diff
@@ -478,3 +478,11 @@ func (l Location) SubLocation() int {
 		return -1
 	}
 }
+
+func (l Location) RPCMarshal() []hexutil.Uint64 {
```

Comment:
Can we change the name of this function to just Marshal?. or just move this function to the quaiapi folder and not as a helper method. Then RPCMarshal makes sense.